kapitan/refs/base.py

```diff
@@ -506,6 +503,24 @@ def ref_from_embedded(self, type_name, b64_path):
 
         return ref
 
+    def ref_from_ref_file(self, ref_file_path):
+        "returns ref from a ref file_path"
+        with open(ref_file_path, "r") as ref_file:
+            ref_file_obj = yaml.safe_load(ref_file)
+
+            type_name = ref_file_obj.pop("type")
+            encoding = ref_file_obj["encoding"]
+            data = ref_file_obj.pop("data")
+            from_base64 = encoding == "base64"
+
+            backend = self._get_backend(type_name)
+
+            # create new ref with deserialised data and remaining keys as kwargs
+            # note that encrypt=False is only for secret ref types, non secret refs (e.g. base64) will ignore
+            # from_base64 is True because data is always base64 encoded in embedded form
```

Comment:
Change comment to "from_base64 is True because data is being loaded from a ref file where it is always base64"